Rovers/Rovers.tex

```diff
@@ -0,0 +1,344 @@
+The MVP is intended to be flexible in implementation depending on funding. The size and materials are not defined because of this. Should the funding allow for it there are many improvements that can be made beyond the MVP. 
+
+\begin{table}[ht!]
+    \caption{Improvements}
+    \centering
+    {\renewcommand{\arraystretch}{1.5}
+    \begin{tabularx}{\linewidth}{LL}
+    \hline
+    \textbf{Feature} & \textbf{Reason} \\  
+    \hline
+    Lidar based CV & Autonomous or partially autonomous navigation for future URC readiness. \\
+    Improved navigation with use of GPS system & Additional navigation options. \\
+    Robotic arm with 3+ degrees of freedom & URC readiness and rover capabilities. \\ 
+    Drill or grasper attachment & Increased capabilities of the robotic arm. \\
+    Soil sample analysis with Archimedes' screw & Scientific possibilities. \\
+    Larger and or faster rover & Able to traverse obstacles of greater magnitude or hold more scientific equipment for future expansion. \\
+    Solar cells & Additional long-term navigation viability and experience. \\
+    Web control interface with GUI & Easier navigation. \\
+    Durable construction & Carbon fiber or similar materials for long term viability. \\
+    On board batteries & Lithium Ion, 18650s or similar battery technology for remote abilities. \\
+    Long range communications & Ability to communicate with the rover at lengths of up to one kilometer allows for URC readiness. \\
+    \hline
+    \end{tabularx}
+    }
+\label{tab:mvp-two}
+\end{table}
+
+\section{Secondary Objectives}
+\label{sec:secondary-obj}
+The study will also investigate the University Rover Competition (URC) as hosted by the Mars Society as a potential long term goal. 
+The team will accomplish this by evaluating how many of the MVP improvements are accomplished. 
+If none of these improvements are not able to be implemented then the team is not near URC ready.
+The competition is held annually and features four very intense competitions. 
+Such a rover would need to be fully self navigating, perform scientific analysis on soil samples, and having a robotic arm capable of fine motor control. 
+Such a project would be among the most ambitious SPEX has ever attempted. 
+These features line up with the improvements section of the MVP.
+The skills learned from this project should be applicable to such a future project. 
+This secondary objective is directly related to the long-term vision of the rover project. 
+
+\section{Benefit to SPEX}
+\label{sec:benefit}
+
+\subsection{Space Exploration}
+Rovers are a huge part of space exploration. 
+There are eight active missions on Mars. 
+Four of those include rovers. 
+To ignore this fact would be to ignore a large part of space exploration.
+Currently RIT SPEX is not involved with rovers.
+It would be beneficial to our members to get some experience in this area.
+
+\subsection{Promotions and Funding}
+A rover would look super good for SPEX at Imagine RIT. 
```

Comment:
rephrase "super good" to something less casual. Maybe "A rover would be a popular demonstration at ImagineRIT"

Reply:
Done